capstone/src/components/User/Adventurer/Preference/Preference.jsx

```diff
@@ -148,21 +61,23 @@ export default function Preference() {
               defaultValue={preference.defaultValue}
               step={preference.step}
               units={preference.units}
-              handleDelete={handleDelete}
+              handleDelete={onDelete}
             />
           ))}
         </Box>
         <HStack justifyContent="center">
-          <FilterMenu
-            inactivePreferences={inactivePreferences}
-            handleAddition={handleAddition}
+          <SelectMenu
+            inactiveItems={inactiveInfo}
+            onAdd={onAdd}
           />
-          <Button colorScheme="blue" width="100px" onClick={onSubmission}>Save</Button>
+          <Button colorScheme="blue" width="100px" onClick={onSubmission}>
```

Comment:
Is there a way to set the Chakra theme globally?

Reply:
Yes! I will look for that!